CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-x.x.x Release notes (yyyy-MM-dd)
```

Comment:
Oops, I'd fixed this locally before I requested review but apparently forgot to push it. Take another look?